django/db/backends/base/schema.py

```diff
@@ -838,6 +838,17 @@ def alter_field(self, model, old_field, new_field, strict=False):
             strict,
         )
 
+    def _field_db_check(self, field, field_db_params):
+        # Check constraints always with the same column name to avoid
+        # recreating constrains when the column is renamed.
```

Comment:
I think "mocked" is the key word there. Maybe...

```suggestion
        # Always check constraints with the same mocked column name to avoid
        # recreating constrains when the column is renamed.
```

Reply:
Updated :+1: